packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -355,6 +401,8 @@ Input.propTypes = {
   defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
   disabled: PropTypes.bool,
   fluid: PropTypes.bool,
+  label: customPropTypes.itemShorthand,
+  labelPosition: PropTypes.oneOf<LabelPosition>(['internal', 'inline']),
```

Comment:
What do you think of renaming `internal` to `inside`?

Reply:
I am ok with it :)